### PR TITLE
[java] add more logging to Selenium Manager

### PR DESCRIPTION
**Pull request by titusfortner**
### Description

* Changes log name to `log` which we want to make the standard in the rest of the project (future PR)
* Add `INFO` that about to use Selenium Manager
* Send `--verbose` to Selenium Manager if logging set to `FINE` or higher
* Add `FINE` for: 
    * Command executed
    * Driver Path being used
    * Any DEBUG lines of log 

### Motivation and Context

We need to be able to get more details about what problems users are having 
Also (at least while in Beta) we want users to know that Selenium Manager is being used.
We may want to change that from `INFO` to `FINE` after beta?

Ideally we have a section on Selenium Manager in our documentation that we can point to for explaining what it is doing and how to get more logging info, etc. But that's a future PR.

### CODE REVIEW

**Comment by titusfortner** [comment]
Ah found the issue for this PR - #11642

File: java/src/org/openqa/selenium/manager/SeleniumManager.java [hunk 1]
```diff
@@ -187,24 +196,34 @@ private String getBrowserBinary(Capabilities options) {
...
-          Arrays.asList(binaryFile.getAbsolutePath(),
-                        "--browser",
-                        options.getBrowserName(),
-                        "--output", "json"));
+        List<String> commandList = new ArrayList<>();
+        commandList.add(binaryFile.getAbsolutePath());
+        commandList.add("--browser");
+        commandList.add(options.getBrowserName());
+        commandList.add("--output");
+        commandList.add("--json");
```
Comment (diemol) [inline java/src/org/openqa/selenium/manager/SeleniumManager.java]:
```suggestion
        commandList.add("json");
```

Comment (diemol) [reply java/src/org/openqa/selenium/manager/SeleniumManager.java]: That is a value for the parameter not a parameter itself.

**Review by diemol — CHANGES_REQUESTED** [review]
Left a suggestion

File: java/src/org/openqa/selenium/manager/SeleniumManager.java [hunk 2]
```diff
@@ -54,11 +55,12 @@
 @Beta
 public class SeleniumManager {
 
-    private static final Logger LOG = Logger.getLogger(SeleniumManager.class.getName());
+    private static final Logger log = Logger.getLogger(SeleniumManager.class.getName());
```
Comment (titusfortner) [inline java/src/org/openqa/selenium/manager/SeleniumManager.java]: We have a lot more `LOG` than `log`, so I think we do want to use `LOG` here, I'll change it back

**Review by diemol — APPROVED** [review]

**Pull request merged**